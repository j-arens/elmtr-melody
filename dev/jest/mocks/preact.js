module.exports = {
    unmountComponent: jest.fn(),
};
