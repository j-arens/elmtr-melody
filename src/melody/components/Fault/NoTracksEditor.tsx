import { h } from 'preact';
const s = require('./style.scss');

export default () => (
    <p class={s.ErrorMsg}>
        No tracks loaded, click here to open the settings in the sidebar.
    </p>
);
