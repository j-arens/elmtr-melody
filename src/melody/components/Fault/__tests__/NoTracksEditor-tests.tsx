import { h } from 'preact';
import { render } from 'preact-render-to-string';
import NoTracksEditor from '../NoTracksEditor';

describe('NoTracksEditor', () => {
    it('renders', () => {
        const component = render(<NoTracksEditor/>);
        expect(component).toMatchSnapshot();
    });
});
