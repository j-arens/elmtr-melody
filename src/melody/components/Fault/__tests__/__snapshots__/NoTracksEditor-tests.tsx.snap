// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoTracksEditor renders 1`] = `<p class="ErrorMsg">No tracks loaded, click here to open the settings in the sidebar.</p>`;
